runtime/gc_base/ObjectAccessBarrier.hpp

```diff
@@ -60,6 +60,10 @@ class MM_ObjectAccessBarrier : public MM_BaseVirtual
 	UDATA _ownableSynchronizerLinkOffset; /** Offset within java/util/concurrent/locks/AbstractOwnableSynchronizer of the ownable synchronizer link field */
 public:
 
+	MMINLINE j9flatobject_t indexableFlatObjectAddress(J9VMThread *vmThread, J9IndexableObject *arrayRef, J9Class *clazz, I_32 destIndex) {
+    	return (j9flatobject_t)(indexableEffectiveAddress(vmThread, arrayRef, destIndex, J9ARRAYCLASS_GET_STRIDE(clazz)));
```

Comment:
needs a tab here

Reply:
also instead of destIndex, it can just be index since this helper will be used for both fetching and storing values